spec/models/user_spec.rb

```diff
@@ -0,0 +1,24 @@
+require 'rails_helper'
+
+RSpec.describe User, type: :model do
+  before :all do
+    @user = User.create(username('test'))
+  end
+
+  it 'is valid and added to users' do
+    expect(User.all).to include @user
+    expect(@user).to be_valid
+  end
+
+  it 'useraname should be correct' do
+    expect(@user.username).to eql 'test'
+  end
+  it 'is not valid without a username' do
+    user = User.create
+    expect(user).to_not be_valid
+  end
+  it 'username must be unique' do
+    user = User.create('test')
```

Comment:
kindly fix this line , a hash should be passed as an argument not a string .

Reply:
I thought it was a typo but I think when there is only one key in a object you don't need the key.. 🤔 
This is due to the fact that a username must be unique I used a  method to generate a unique string each time.